challenges/prototypes.js

```diff
@@ -6,28 +6,47 @@
   Create a constructor function named CuboidMaker that accepts properties for length, width, and height
 */
 
+function CuboidMaker(cuboidMakerAttrs) {
+  this.length = cuboidMakerAttrs.length;
+  this.width = cuboidMakerAttrs.width;
+  this.height = cuboidMakerAttrs.height;
+}
```

Comment:
You don't need to add cubioMakerAttrs assigned to each this. Still though, good work.